### PR TITLE
Made visual updates to Rated Disabilities page

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26181)

Upon design review, it was determined that we should get rid of the sidebar on the Rated Disabilities page and move some of that content into the main area of the page. It was also determined that when there are rated disabilities that the page gets too long and we should add anchor links on the page. This PR accomplishes these changes.

## Testing done
All existing unit tests still pass

## Screenshots

screencapture-localhost-3001-disability-view-disability-rating-rating-2021-07-01-12_18_25

## Acceptance criteria
- [x] The feature has been synced with latest mockups
- [x] A PR has been requested for merging

### CODE REVIEW

File: src/applications/personalization/rated-disabilities/components/OnThisPageLink.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const OnThisPageLink = props => {
+  return (
+    <a className="arrow-down-link" href={props.link}>
+      <p>
+        <span className="icon-with-info">
+          <i className="fa fa-arrow-down vads-u-padding-right--1" />
```
[the remark this answers is not in the document]
Comment (timwright12) [reply src/applications/personalization/rated-disabilities/components/OnThisPageLink.jsx:8]: @cohnjesse noting that this is a draft, but I wanted to make sure adding `aria-hidden="true"` to this icon didn't slip through the cracks

File: src/applications/personalization/rated-disabilities/components/RatedDisabilityView.jsx [hunk 2]
```diff
@@ -81,9 +116,29 @@ class RatedDisabilityView extends React.Component {
...
+              >
+                Learn about VA disability ratings
+              </h3>
+              <p>
+                To learn how we determined your VA combined disability rating,
+                use our disability rating calculator and ratings table.
+              </p>
+              <a
+                href="/disability/about-disability-ratings/"
+                aria-label="About VA disability ratings"
```
Comment (timwright12) [inline src/applications/personalization/rated-disabilities/components/RatedDisabilityView.jsx:131]: No need for the aria-label if it matches the link text

Comment (cohnjesse) [reply src/applications/personalization/rated-disabilities/components/RatedDisabilityView.jsx]: @timwright12 Good catch! Removed this

File: src/applications/personalization/rated-disabilities/components/OnThisPageLink.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (micahchiang) [reply src/applications/personalization/rated-disabilities/components/OnThisPageLink.jsx:8]: @cohnjesse when you have a sec, could you add `aria-hidden="true"` here? Thanks!

**Review by micahchiang — APPROVED** [review]
LGTM. Please add `aria-hidden="true"` to the icon before merging 👍

File: src/applications/personalization/rated-disabilities/components/OnThisPageLink.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/personalization/rated-disabilities/components/OnThisPageLink.jsx:8]: Added this

**Pull request merged**